tests/ignite/contrib/metrics/regression/test_mean_absolute_relative_error.py

```diff
@@ -76,3 +78,46 @@ def test_zero_sample():
     m = MeanAbsoluteRelativeError()
     with raises(NotComputableError, match=r"MeanAbsoluteRelativeError must have at least one sample"):
         m.compute()
+
+
+def test_integration():
+    def _test(y_pred, y, batch_size):
+        def update_fn(engine, batch):
+            idx = (engine.state.iteration - 1) * batch_size
+            y_true_batch = np_y[idx : idx + batch_size]
+            y_pred_batch = np_y_pred[idx : idx + batch_size]
+            return idx, torch.from_numpy(y_pred_batch), torch.from_numpy(y_true_batch)
+
+        engine = Engine(update_fn)
+
+        m = MeanAbsoluteRelativeError(output_transform=lambda x: (x[1], x[2]))
+        m.attach(engine, "mare")
+
+        np_y = y.numpy()
+        np_y_pred = y_pred.numpy()
+
+        data = list(range(y_pred.shape[0] // batch_size))
+        mare = engine.run(data, max_epochs=1).metrics["mare"]
+
+        abs_error = np.sum(abs(np_y - np_y_pred) / abs(np_y))
+        num_samples = len(y_pred)
+        sum_error = abs_error
+        sum_samples = num_samples
+        res = sum_error / sum_samples
```

Comment:
```suggestion
        res = abs_error / num_samples
```
and remove `sum_samples` and `sum_error`